jsk_kinova_robot/jsk_kinova_description/robots/gen3_lite_gen3_lite_2f.xacro

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0"?>
+<robot name="jsk_gen3_lite_gen3_lite_2f" xmlns:xacro="http://ros.org/wiki/xacro">
+    <!-- Arguments -->
+    <xacro:arg name="arm" default="gen3_lite" />
+    <!-- Those arguments are ignored for now on Gen3 lite because there is only one supported configuration -->
+    <xacro:arg name="dof" default="6" />
+    <xacro:arg name="vision" default="false" />
+    <xacro:arg name="gripper" default="gen3_lite_2f" />
+    <xacro:arg name="sim" default="false" />
+    <xacro:arg name="prefix" default="" />
+
+    <xacro:include filename="$(find kortex_description)/robots/kortex_robot.xacro" />
+    <xacro:load_robot arm="$(arg arm)" gripper="$(arg gripper)" dof="$(arg dof)" vision="false" sim="$(arg sim)"  prefix="$(arg prefix)" />
+
+    <link name="kinova_wrist_camera_link">
```

Comment:
can you explain why you need this link. Did you consider publishing via 
`static_transform_publisher` c.f. https://github.com/ros-drivers/rgbd_launch/blob/indigo-devel/launch/kinect_frames.launch

Reply:
I think this will be necessary when collision between robot links and camera happens.

Reply:
@sktometometo this link does not includes geometry, so it is not calculate the collision.

Reply:
I will add camera mount model to this link after our kinova arm sent back.